ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/KtorCallContexts.kt

```diff
@@ -86,3 +86,11 @@ public class OnCallRespondContext<PluginConfig : Any> internal constructor(
         context.subject = transformContext.transform(context.subject)
     }
 }
+
+public suspend fun <T : Any> PipelineContext<T, ApplicationCall>.transformBody(
```

Comment:
does this function needs to be public?

Reply:
is it mean that I can annotate it with `@InternalAPI` and it in plugins write `@OptIn(InternalAPI::class)`?

Reply:
oh, nevermind, I read the next comment :)